SU2_CFD/include/fluid/CSutherland.hpp

```diff
@@ -39,47 +39,47 @@ class CSutherland final : public CViscosityModel {
   /*!
    * \brief Constructor of the class.
    */
-  CSutherland(su2double mu_ref, su2double t_ref, su2double s) : mu_ref_(mu_ref), t_ref_(t_ref), s_(s) {}
+  CSutherland(su2double mu_ref, su2double t_ref, su2double s) : mu_ref(mu_ref), t_ref(t_ref), s(s) {}
 
   /*!
    * \brief return viscosity value.
    */
-  su2double GetViscosity() const override { return mu_; }
+  su2double GetViscosity() const override { return mu; }
 
   /*!
    * \brief return viscosity partial derivative value.
    */
-  su2double Getdmudrho_T() const override { return dmudrho_t_; }
+  su2double Getdmudrho_T() const override { return dmudrho_t; }
 
   /*!
    * \brief return viscosity partial derivative value.
    */
-  su2double GetdmudT_rho() const override { return dmudt_rho_; }
+  su2double GetdmudT_rho() const override { return dmudt_rho; }
 
   /*!
    * \brief Set Viscosity.
    */
   void SetViscosity(su2double t, su2double rho) override {
-    const su2double t_nondim = t / t_ref_;
-    mu_ = mu_ref_ * t_nondim * sqrt(t_nondim) * ((t_ref_ + s_) / (t + s_));
+    const su2double t_nondim = t / t_ref;
```

Comment:
Do you know why the trailing "_" was put for these variables?

Reply:
To be honest, I do not know. Should I keep them?

Reply:
I don't know either haha

Reply:
Yes